working-groups/README.md

```diff
@@ -1,7 +1,47 @@
 # Working Groups
 
-Each AMP Working Group is a segment of the community with knowledge/interest in specific area of AMP, with a mandate provided by the Technical Steering Committee.  See [GOVERNANCE.md](../GOVERNANCE.md#working-groups) for more details.
 
-## List of working groups
+An AMP [Working Group](https://github.com/ampproject/meta/blob/master/GOVERNANCE.md#working-groups) is a segment of the community with knowledge/interest in specific area of AMP.  Working Groups are created by AMP’s [Technical Steering Committee](https://github.com/ampproject/meta/blob/master/GOVERNANCE.md#technical-steering-committee-tsc). 
+
+AMP’s Working Groups are:
```

Comment:
Add links to repos

Reply:
I'll add these in a future PR once we have the repos set up.  Did we come to a conclusion on separate repos vs. one repo?

Reply:
Sounds like we're going with one repo per group.